### PR TITLE
Reintroduces refined and unrefined scrap, changes baseline for walls

### PR DESCRIPTION
**Pull request by zombiemimic**
## About The Pull Request
Adds in Unrefined scrap, which can be refined for Refined scrap and other bits of materials. As a bonus, walls made with abnormal materials now look normal.

## Why It's Good For The Game
putting preexistent code to good use

## Changelog
:cl:
add: Added recyclable 'Unrefined Scrap' to the trash piles
fix: fixed walls having incongruous textures
/:cl:

### CODE REVIEW

File: code/modules/scrap/scrap.dm [hunk 1]
```diff
@@ -312,7 +312,8 @@ GLOBAL_LIST_EMPTY(scrap_base_cache)
 		dig_amount--
 		for (var/a in matter)
 			matter[a] *=RAND_DECIMAL(0.6, 0.8)//remove some amount of matter from the pile
-		//new /obj/item/scrap_lump(src) //Todo: uncomment this once purposes and machinery for scrap are implemented
+		if(prob(55))
+			new /obj/item/trash/scrap_lump(src) //Todo: uncomment this once purposes and machinery for scrap are implemented
```
Comment (TheShown911) [inline code/modules/scrap/scrap.dm:316]: You can remove the TODO here

File: code/modules/scrap/scrap_refine.dm [hunk 2]
```diff
@@ -25,28 +25,37 @@
...
-		reagents.add_reagent(pick(GLOB.chemical_reagents_list), rand(1, reagents.maximum_volume))
+		reagents.add_reagent(pick(GLOB.chemical_reagents_list), rand(1, reagents.maximum_volume))*/
 	pixel_x = rand(0, 16) - 8
 	pixel_y = rand(0, 8) - 8
+	matter = list(	MATERIAL_STEEL = rand(0, 2.5),
+					MATERIAL_WOOD = rand(0, 1.5), 
+					MATERIAL_GLASS = rand(0, 1.5), 
+					MATERIAL_PLASTIC = rand(0, 2.5),
+					MATERIAL_PLASTEEL = rand(0, 0.75),
+					MATERIAL_URANIUM = rand(0, 0.5),
```
Comment (TheShown911) [inline code/modules/scrap/scrap_refine.dm:50]: Not sure about uranium here.

**Review by TheShown911 — COMMENTED** [review]
Would like to see more uses for this except walls and power.

**Comment by drexample** [comment]
Is this a TF2 reference?

File: code/modules/scrap/scrap_refine.dm [hunk 3]
```diff
@@ -25,28 +25,36 @@
...
+					MATERIAL_PLASTIC = rand(0, 2.5),
+					MATERIAL_PLASTEEL = rand(0, 0.75),
+					MATERIAL_REFSCRAP = 1
+					)
 
-/obj/item/stack/refined_scrap
+/obj/item/stack/material/refined_scrap
 	name = "refined scrap"
-	desc = "This is ghetto gold! It could be used as fuel or building material. Even central Command would give cargo points for this."
+	desc = "This is ghetto gold! It could be used as fuel or sold abroad."
```
Comment (TheShown911) [inline code/modules/scrap/scrap_refine.dm:55]: I don't think the term abroad is applicable in a space ship.

Comment (zombiemimic) [reply code/modules/scrap/scrap_refine.dm:55]: perhaps not, but I deleted my fork a while back, so I can't change it

Comment (vode-code) [reply code/modules/scrap/scrap_refine.dm:55]: remember, this is why you don't delete your stuff before the PR is merged, especially when said stuff is a fork.
The solutions I see are:
1. edit via maintainer access to PRs(uncertain if will work)
2. re-fork and restore code from VSC if you didn't delete your local
3. edit this in another PR

If this is going to be edited, I'd suggest changing it from "sold abroad" to "exported".

**Pull request closed without merging**